src/uwtools/rocoto.py

```diff
@@ -26,6 +31,57 @@ def _add_jobname(tree: dict) -> None:
 
 
 # Public functions
+def realize_rocoto_xml(
+    input_yaml: str,
+    input_template: str,
+    rendered_output: str,
+    schema_file: str,
```

Comment:
Why not hard-code the schema file in here so any user of this code gets that benefit. Then it doesn't need to be tracked in both places?

The same is applicable for the Rocoto JSONSchema.

Reply:
I debated this, whether to leave the function universal or not, but I think you're right in cementing it here